tests/utils_tests/test_inspect.py

```diff
@@ -103,6 +114,17 @@ def test_func_accepts_kwargs(self):
         self.assertIs(inspect.func_accepts_kwargs(Person.all_kinds), True)
         self.assertIs(inspect.func_accepts_kwargs(Person().just_args), False)
 
+    def test_regression_36696(self):
```

Comment:
Use descriptive name instead of the ticket number, e.g.
```suggestion
    def test_func_accepts_kwargs_deferred_annotations(self):
```

Reply:
Fixed, thank you